task/rhacs-networkpolicy-generate/3.73/rhacs-networkpolicy-generate.yaml

```diff
@@ -0,0 +1,112 @@
+---
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  name: rhacs-networkpolicy-generate
+  labels:
+    app.kubernetes.io/version: "3.73"
```

Comment:
can you please follow semver here?

Reply:
The task leverages 3.73.x of the rhacs tool and won't rev to track patches. It will rev at release of rhacs 3.74. The other existing rhacs tasks use eg. `3.71`.

The example in https://github.com/tektoncd/catalog/blob/main/README.md and the other tasks in the catalog which I surveyed have false major.minor versions like `0.1`, not semver `x.y.z`.